src/form.js

```diff
@@ -353,6 +353,16 @@ define([
                 return m.p.nodeID === nodeID;
             });
         },
+        getMugChildrenByChoiceValue: function (mug, nodeID) {
+            var parentNode = (mug ? this.dataTree.getNodeFromMug(mug)
+                                  : this.dataTree.rootNode);
+            return _.filter(parentNode.value._node_control.children, function (m) {
```

Comment:
D'oh. I tried too hard to model it after  getMugChildrenByNodeID.

Yours is much better